src/instrument/devices/local_controls.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Looks like a util

Reply:
I think I agree with that.

Reply:
Probably need a more descriptive name when it moves to `utils`. Maybe, `instrument_class`

Reply:
done: `utils.yaml_devices`